ambari/datadog_checks/ambari/data/conf.yaml.example

```diff
@@ -119,6 +119,50 @@ instances:
 #
 # logs:
 #   - type: file
-#     path: /var/log/ambari-server/*.log
+#     path: /var/log/ambari-server/ambari-alerts.log
+#     source: ambari
+#     service: ambari
+#     log_processing_rules:
+#        - type: multi_line
+#          name: new_log_start_with_date
+#          pattern: \d{4}\-(0?[1-9]|1[012])\-(0?[1-9]|[12][0-9]|3[01])  # 2019-04-22 15:47:00,999
+#   - type: file
+#     path: /var/log/ambari-server/ambari-audit.log
+#     source: ambari
+#     service: ambari
+#     log_processing_rules:
+#        - type: multi_line
+#          name: new_log_start_with_date
+#          pattern: \d{4}\-(0?[1-9]|1[012])\-(0?[1-9]|[12][0-9]|3[01])  # 2019-05-25T13:13:17.875Z
+#   - type: file
+#     path: /var/log/ambari-server/config-changes.log
+#     source: ambari
+#     service: ambari
+#   - type: file
+#     path: /var/log/ambari-server/ambari-eclipselink.log
+#     source: ambari
+#     service: ambari
+#   - type: file
+#     path: /var/log/ambari-server/server-check-database.log
+#     source: ambari
+#     service: ambari
+#   - type: file
+#     path: /var/log/ambari-server/ambari-server-command.log
+#     source: ambari
+#     service: ambari
+#     log_processing_rules:
+#        - type: multi_line
+#          name: new_log_start_with_date
+#          pattern: (INFO|ERROR|WARN)  # ERROR 2019-04-22
```

Comment:
```suggestion
#          pattern: [A-Z]+ \d{4}-\d{2}-\d{2}   # ERROR 2019-04-22
```
This seems not reliable since the log severity may contain something else than:  `INFO|ERROR|WARN`

Reply:
What does \d{4}-\d{2}-\d{2} mean?

Reply:
`\d{4}-\d{2}-\d{2}` will match `2019-04-22` like here https://github.com/DataDog/integrations-core/blob/master/gunicorn/datadog_checks/gunicorn/data/conf.yaml.example#L52

This would work too, more precise, but a bit overkill IMO: https://github.com/DataDog/integrations-core/blob/master/kafka/datadog_checks/kafka/data/conf.yaml.example#L115